client/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html ng-app='GitSecure'>
+  <head>
+    <meta charset="utf-8" />
+        <script src="lib/bower_components/angular/angular.js"></script>
```

Comment:
This looks like you are referencing bower components but we do not have any bower.json file. If you installed with bower please be sure you did a `bower init` from the root of the project first and the `bower --save module` for each module you installed. This will add a bower_components folder to root which should be added to .gitignore. You would then need to update these script tags to point to the new location. We can setup a .bowerrc if it makes more sense to move bower components into the client folder. This may also have implications to our current Gruntfile.js setup (ensuring that bower_componets are not being watched or linted).